web/src/main/java/com/futurewei/alcor/web/entity/route/InternalRouterConfiguration.java

```diff
@@ -38,6 +38,9 @@
     @JsonProperty("host_dvr_mac")
     private String host_dvr_mac;
 
+    @JsonProperty("default_routing_table")
+    private List<InternalRoutingRule> default_routing_table;
+
```

Comment:
we don't use `default_routing_table` anymore.

Reply:
Removed